csuntechlab/iSTART
# STR-71 -- FE: Hamburger Menu does not fill up all the way at once

# Description
Added event to toggle a new dropdown navigation.
  
# Testing
1. Login as any user ***`example: cyndee.bellamy`*** .
2. Ensure the top navigation bar is properly aligned to content.
3. Click the navigation button and ensure the dropdown functionality works.
4. Make sure everything looks good and has no weird visual bugs.
5. Ensure the visual quality at different viewports.
